# Compressed Image Array Message
# This message contains an array of compressed images

sensor_msgs/CompressedImage stopping_img
sensor_msgs/CompressedImage[] defects
